libmscore/measurenumber.cpp

```diff
@@ -57,6 +57,9 @@ QVariant MeasureNumber::propertyDefault(Pid id) const
 
 void MeasureNumber::layout()
       {
+      if (!measure())
```

Comment:
The error was actually happening on line 69 (just below). 
First used `staff()->constStaffType()` and only afterwards verified that `staff()` actually existed. You can see that in the code from line 70 to 75.
My fix to this bug only included that:
```
      if (!parent()) {
            setOffset(0.0, 0.0);
            return;
            }
```

Reply:
Actually, I was wrong, as `TextBase::layout1()` still needs to be called.
See [this implementation](https://github.com/musescore/MuseScore/compare/master...Marr11317:mn-palette?expand=1#diff-c6d784b581b8be852aa62d1fc8e37770) for details.